node/src/components/consensus/consensus_protocol.rs

```diff
@@ -204,4 +204,6 @@ pub(crate) trait ConsensusProtocol<I, C: Context>: Send {
     /// Returns the protocol outcomes for all the required timers.
     /// TODO: Remove this once the Joiner no longer has a consensus component.
     fn recreate_timers(&self) -> Vec<ProtocolOutcome<I, C>>;
+
+    fn next_round_length(&self) -> Option<TimeDiff>;
```

Comment:
Let's add a TODO and create a story in [FEAT-274](https://casperlabs.atlassian.net/browse/FEAT-274) to find a way to report Highway-specific details without making this trait Highway-specific.

Reply:
Done in [b23a825](https://github.com/CasperLabs/casper-node/pull/954/commits/b23a825ab7e68d4c32ee7e6ae0c449cdefac674d).

Reply:
https://casperlabs.atlassian.net/browse/HWY-260